src/test/resources/application.yml

```diff
@@ -25,6 +25,9 @@ kestra:
       table: "${random.shortuuid}"
     firestore:
       project: "kestra-unit-test"
+    monitoring:
+      project: "kestra-unit-test"
+      resourceType: "global"
```

Comment:
I think we need to add some roles in the `infra` repository for Google Monitoring here to make the CI pass right?